<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SliderWidget</class>
 <widget class="QWidget" name="SliderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QToolButton" name="addButton">
     <property name="text">
      <string>Add</string>
     </property>
     <property name="icon">
      <iconset theme="list-add">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>382</width>
        <height>253</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="slidersLayout"/>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>addButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
